src/scf-release/src/acceptance-tests-brain/test-scripts/minibroker_helper.rb

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env ruby
+
+require_relative 'testutils'
+require 'json'
+require 'timeout'
+
+module M
+    refine Module do
+        def attr_lazy(name, &block)
+            sym = "@#{name}".to_sym
+            self.send(:define_method, name) do
+                self.instance_variable_set sym, block.call(self) unless self.instance_variable_defined? sym
+                self.instance_variable_get sym
+            end
+        end
+    end
+end
+
+using M
+
+# MiniBrokerTest is a helper class to test minibroker-provided services
```

Comment:
[nit] Missing punctuation here and below.

Reply:
Fixed in 9212de175877a3e8df5d69b069037df9af3bc712